CHANGELOG.rst

```diff
@@ -1,6 +1,10 @@
 Change Log
 ==========
 
+v4.0.3 (UNRELEASED)
```

Comment:
Don't forget to update release date and also fix the unit tests. They likely fail now due to the removal of 'order'.

Reply:
fixed that!